### ISSUE TITLE
[Request] Subtitle Support

### ISSUE THREAD

**Issue by github-throwaway** [ram-on/SkyTube#89]
I would love to see support for subtitles/closed captions.

**Comment by dimqua** [comment]
Any news? Is it planned any time soon?

**Comment by ram-on** [comment]
Currently you can enable subtitles if you use SkyTube Extra > go to preference > use Official player.

In the future we might add support to YouTube's official HTML5 player which will be included in both SkyTube and SkyTube Extra.  That would resolve this issue.

**Comment by dimqua** [comment]
@ram-on is it possible to include YouTube's official player to SkyTube Extra and remove the need of YouTube app installed? Since YouTube application itself doesn't work without Google Play Services, but official player does.

**Comment by ram-on** [comment]
@dimqua Nope.  YouTube's official player (as a library) depends on YouTube's official app.

**Comment by Caboosium** [comment]
I thought youtube was factory software on most phones but I guess I did say "most"...

**Comment by anishmittal2020** [comment]
@ram-on Youtube official player shows add and if we cast it over chromecast using the official player, the subtitles vanish.
